src/Relay.ts

```diff
@@ -0,0 +1,78 @@
+interface Message {
+  to?: string | undefined
+  message: string
+  payload?: any | undefined
```

Comment:
There's a slight difference between an optional (`?:`) field and one that can be `undefined`. There are definitely use cases for each style, but you don't usually need to use both together. In this case, I would say that the optional fields don't need `| undefined`, since that's implied by the optionality. Also, `any` is already a sinkhole for type-safety, so `any | undefined` might as well be `any`.

Here's my recommendation:
```ts
interface Message<TPayload = any> {
  to?: string;
  message: string;
  payload?: TPayload;
}
```